### PR TITLE
README update to address issue #113

### PR DESCRIPTION
**Pull request by sparrc**
\cc @beckettsean @toddboom I'm updating the README to clarify InfluxDB/Telegraf version compatibility, let me know if you think this needs to be explained better!

see the README here: https://github.com/influxdb/telegraf/tree/cs-113

### CODE REVIEW

**Comment by sparrc** [comment]
ps: you can see the README here: https://github.com/influxdb/telegraf/tree/cs-113

File: README.md [hunk 1]
```diff
@@ -23,10 +23,19 @@ agent is NOT backwards-compatible with a config file from 0.1.4 and below.
...
 
-```
-http://get.influxdb.org/telegraf/telegraf_0.1.5_amd64.deb
-http://get.influxdb.org/telegraf/telegraf-0.1.5-1.x86_64.rpm
-```
+As well, due to a breaking change to the InfluxDB integer line-protocol, there
+are some InfluxDB compatibility requirements:
+
+* InfluxDB <= 0.9.2 requires Telegraf <= 0.1.4
+* InfluxDB > 0.9.2 requires Telegraf > 0.1.4 (this includes any nightly builds)
```
Comment (beckettsean) [inline README.md]: I would invert these, and convert them to our version styles (very new addition, not even remotely consistent yet)

InfluxDB 0.9.3+ (including nightly builds) requires Telegraf 0.1.5+ 
InfluxDB 0.9.2 and prior requires Telegraf 0.1.4 

I don't think we need to reference earlier versions of Telegraf. Is there a reason to use anything but 0.1.4 with InfluxDB 0.9.0 through 0.9.2?

**Comment by beckettsean** [comment]
Minor style note, otherwise :+1:

File: README.md (hunk 1 above)
Comment (sparrc) [inline README.md]: Cool, I'll update to reflect that! And as far as I know there is no reason to use Telegraf versions before 0.1.4

**Pull request merged**